### DISCUSSION TITLE
[Feature] Include sharing of Albums with Partners

### DISCUSSION THREAD

**Discussion by sigulete** [immich-app/immich#7026] [Feature Request]
### The feature

When you add partners to your account, they can see your photos, which is a great start. But I would like to request for partners to see the albums as well.

In my case, my wife is the one who takes photos, and I'm the one who upload them, create albums, name them, and set the album covers. 
Ideally, I would like for my wife and kids to access all Photos and Albums, so everyone benefit from my hard work :)

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by MikeBishop** [comment]
Same here. It's possible to share albums one by one, but it would be helpful to default albums to shared and/or be able to share many albums at once.
